vortex-io/src/lib.rs

```diff
@@ -18,10 +19,9 @@ pub use read_ranges::*;
 pub use tokio::*;
 pub use write::*;
 
-mod aligned;
 mod buf;
-#[cfg(feature = "compio")]
-mod compio;
+// #[cfg(feature = "compio")]
```

Comment:
@a10y do you mind if I do this for now? Once we move VortexBuffer over to be aligned, then we can impl compio traits. But I want to get develop fixed by aliasing aligned_buffer for now instead of wrapping it and re-exposing the whole API just so we can impl the compio traits.